pilot/pkg/model/config_test.go

```diff
@@ -175,6 +175,38 @@ func TestServiceKey(t *testing.T) {
 	}
 }
 
+func TestBuildSubsetKey(t *testing.T) {
```

Comment:
It might be good to write a quick test that makes sure that `BuildSubsetKey` and `ParseSubsetKey` are inverses of each other. Basically:
```
ParseSubsetKey(BuildSubsetKey(xxx)) == xxx
```